dotCMS/src/main/java/com/dotcms/system/event/local/type/staticpublish/SingleStaticPublishEndpointSuccessEvent.java

```diff
@@ -0,0 +1,42 @@
+package com.dotcms.system.event.local.type.staticpublish;
+
+import com.dotcms.publisher.endpoint.bean.PublishingEndPoint;
+import com.dotcms.publishing.PublisherConfig;
+import com.dotcms.system.event.local.type.publish.PublishEvent;
+import java.util.Date;
+
+/**
+ *  Object used to represent an event to be triggered when an endpoint successes during static publishing
+ *
+ * @author nollymar
+ */
+public class SingleStaticPublishEndpointSuccessEvent extends PublishEvent {
+
+    private PublisherConfig config = null;
+    private PublishingEndPoint endpoint;
+
+    public SingleStaticPublishEndpointSuccessEvent(PublisherConfig config, PublishingEndPoint endpoint) {
+        setName(AllStaticPublishEndpointsSuccessEvent.class.getCanonicalName());
+        setPublishQueueElements(config.getAssets());
+        setDate(new Date());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the Java 8 Date and Time API instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2143)